.github/workflows/release.yml

```
@@ -32,7 +32,7 @@ jobs:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
       - name: Setup Node.js with GitHub Package Registry
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2.1.5
```

So you don't get too much PRs every time a patch version is released

```suggestion
        uses: actions/setup-node@v2
```